app/static/app/js/Console.jsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import './css/Console.scss';
+import './vendor/google-code-prettify/prettify';
+import './vendor/google-code-prettify/prettify.css';
+import update from 'react-addons-update';
+import Utils from './classes/Utils';
+
+class Console extends React.Component {
```

Comment:
I was playing around with [xterm.js](https://github.com/sourcelair/xterm.js) yesterday. It doesn't eliminate the need for this component, but it may be a nice addition, especially since it does a nice job of display ANSI escape codes (which are present in the ODM log output).

Improved auto-scrolling behavior is being tracked in https://github.com/sourcelair/xterm.js/pull/336 and is scheduled to land in next Monday's release.